prow/tide/tide.go

```diff
@@ -1065,10 +1112,24 @@ func (c *Controller) syncSubpool(sp subpool, blocks []blockers.Blocker) (Pool, e
 			Action:   act,
 			Target:   targets,
 			Blockers: blocks,
+			Error:    errorString,
 		},
 		err
 }
 
+func prMeta(prs ...PullRequest) []history.PRMeta {
+	res := make([]history.PRMeta, 0, len(prs))
```

Comment:
```go
var res []history.PRMeta
```

Should be fine?

Reply:
You're right -- this comment and the others were more about less complication. Not sure we we worried about performance quite yet. Nit either way.